lib/govuk_publishing_components/presenters/checkboxes_helper.rb

```diff
@@ -4,7 +4,7 @@ class CheckboxesHelper
       include ActionView::Helpers
       include ActionView::Context
 
-      attr_reader :items, :name, :css_classes, :error, :has_conditional, :has_nested, :id, :hint_text
+      attr_reader :items, :name, :css_classes, :error, :has_conditional, :has_nested, :id, :hint_text, :no_hint_text
```

Comment:
Is `no_hint_text` used outside this class? If not, I don't think this is needed.